patched-vscode/extensions/code-editor-idle-extension/README.md

```diff
@@ -0,0 +1,3 @@
+# Code Editor Idle Extension
+
+The Code Editor Idle Extension tracks user activity and logs the last active timestamp (in UTC) to a local file. User activities monitored include file changes, cursor movements, and terminal interactions. Additionally, it provides an API endpoint /api/idle that returns the lastActiveTimestamp.
```

Comment:
where are we checking for `cursor movements` ?

Reply:
there is an event listener on `vscode.window.onDidChangeTextEditorSelection` which triggers whenever the editor cursor moves.

Reply:
Oh are you referring to mouse cursor movements? If so, we're not planning to track mouse movements, here I meant the text editor cursor movement. Ill update the text to be more explicit like "changes to text editor selections".